bin/drishti_build.cmd

```diff
@@ -17,7 +17,7 @@ python %POLLY_ROOT%\bin\polly.py ^
 --config "%1%" ^
 --toolchain "%2%" ^
 --test ^
---fwd HUNTER_USE_CACHE_SERVERS=NO ^
+--fwd HUNTER_USE_CACHE_SERVERS=YES ^
 HUNTER_DISABLE_BUILDS=NO ^
```

Comment:
`HUNTER_USE_CACHE_SERVERS=ONLY` `HUNTER_DISABLE_BUILDS=YES`?

Reply:
This script was added based on issue discussions to provide an easy option for local host builds.  Once cmake defaults + documentation is cleaned up I think it can be removed (along with some others).  It isn't actually used right now in appveyor.yml.  The appveyor `build-appveyor.cmd` script used in CI still forces cache behavior:

 `- cmd: bin\build-appveyor.cmd "%CONFIG%" "%TOOLCHAIN%"`

Reply:
ok